main.js

```diff
@@ -0,0 +1,6 @@
+let message ="hi this is my homework" 
```

Comment:
Spacing and semicolon
```suggestion
let message = "hi this is my homework";
```